### PR TITLE
Fix empty namespace for plugin in link creation

### PR DESCRIPTION
**Pull request by Kerruba**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: src/main/java/org/entando/kubernetes/service/EntandoLinkService.java [hunk 1]
```diff
@@ -6,22 +6,23 @@
...
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Slf4j
 @Service
-public class AppPluginLinkService {
+public class EntandoLinkService {
 
-    private final @NonNull KubernetesClient client;
+    private final KubernetesClient client;
```
Comment (joewhite101) [inline src/main/java/org/entando/kubernetes/service/EntandoLinkService.java]: Annotation dropped on purpose?

Comment (Kerruba) [reply src/main/java/org/entando/kubernetes/service/EntandoLinkService.java]: We can restore that probably if we want

**Pull request merged**